### PR TITLE
[zh-cn]: create the translation of `MediaDevices.selectAudioOutput()` method

### PR DESCRIPTION
**Pull request by tianyeeT**
https://github.com/mdn/translated-content/issues/20744#issuecomment-2163722099

### CODE REVIEW

**Comment by yin1999** [comment]
Blocked by mdn/content#34088

**Review by skyclouds2001 — COMMENTED** [review]
https://github.com/mdn/content/pull/34088 has been merged

File: files/zh-cn/web/api/mediadevices/selectaudiooutput/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "MediaDevices: selectAudioOutput() method"
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/mediadevices/selectaudiooutput/index.md:2]:
```suggestion
title: MediaDevices：selectAudioOutput() 方法
```

File: files/zh-cn/web/api/mediadevices/selectaudiooutput/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        > 如果先前授予了指定设备的权限，但此后被撤销，用户代理可能会显示所有允许的设备，并突出显示具有指定 ID 的设备。
+
+### 返回值
+
+{{ jsxref("Promise") }}，其会兑现一个描述用户选择的音频输出设备的 {{domxref("MediaDeviceInfo")}} 对象。
+
+### 异常
+
+- `NotAllowedError` {{domxref("DOMException")}}
+  - : 如果使用[`speaker-selection`](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection) [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)来阻止使用音频输出（此外不会显示用于选择音频输出的弹出窗口），或者用户关闭选择提示而没有选择设备时，会返回此错误。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/mediadevices/selectaudiooutput/index.md:46]:
```suggestion
  - : 如果使用 [`speaker-selection`](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection) [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)来阻止使用音频输出（此外不会显示用于选择音频输出的弹出窗口），或者用户关闭选择提示而没有选择设备时，会返回此错误。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**